xml/obs_admin_components.xml

```diff
@@ -76,6 +76,14 @@
     directory on the back-end. Each OBS installation has one publisher per
     partition. </para>
   </sect3>
+  <sect3>
+   <title>Source Publisher</title>
+   <para> The source publisher processes "sourcepublish" events from the publisher for
+    finished repositories. It needs to run on the same server as the source server.
```

Comment:
* "finished repository" sounds wrong. Also not 100% sure what is meant -- "a repository in which all jobs have finished building"? 
* I guess this is a bigger terminology issue -- but using "server" for two different things (software component v/ machine) in the same sentence is a bit confusing.

Reply:
using "instance" now